### ISSUE TITLE
best way to troubleshoot api key failures?

### ISSUE THREAD

**Issue by ceteris** [apptreesoftware/flutter_google_map_view#99]
I enabled all the api's, edited all the config files, etc, still getting a 403 when the example tries to load the static image

maybe somebody has a barebones test app I can run?

thanks

**Comment by shoxter** [comment]
I'm having this same issue.

I didn't change any of my code, it was working flawlessly and now I'm getting 403s.

The URL is correct and the parameters are as expended.

I think it's an API related issue?

**Comment by ceteris** [comment]
I'm still stumped. I wrote a simple android/java app to troubleshoot and it works with the same API key.

**Comment by shoxter** [comment]
Yeah, I decided to re-implement the aspects of this plugin that I need. It was honestly more trouble than it was work.

I got it working and soon found out that the zoom property on StaticMaps doesn't work if you pass in markers. 

It does work with Google's API. At that point, I opted for reimplementation.

**Comment by ceteris** [comment]
nice. will you be posting your work?

**Comment by shoxter** [comment]
Currently the work I reimplemented is minimal so it isn't worth posting. However, our app is fairly large in size and scope and depending on what additional aspects of Google Maps I reimplement, it may make it worth the time to publish it.

It's just hard to tell right now because our app was previously in Cordova and I've just started redoing it in Flutter over the last few weeks. I can let you know if I do -- how is that?

**Comment by ceteris** [comment]
awesome! thanks

**Comment by marianoarga** [comment]
Same problem here

**Comment by marianoarga** [comment]
The only way I got this working *only when I press the map container* (wich has "You are supposed to see a map here" text) was removing restriction (url firm):
"Note: For development, you can create an unrestricted API key that can be used on both iOS & Android. For production it is highly recommended that you restrict."

Stil the map container has that text and could not display the "inline" map.

**Comment by farukzahra** [comment]
403 here, the url work perfect on browser.
